api/src/test/java/org/openmrs/module/ModuleUtilTest.java

```diff
@@ -187,6 +187,72 @@ public void matchRequiredVersions_shouldAllowRequiredVersionWithWildCard() {
 		Assert.assertTrue(ModuleUtil.matchRequiredVersions(openmrsVersion, requiredOpenmrsVersion));
 	}
 	
+	/**
+	 * @see org.openmrs.module.ModuleUtil#matchRequiredVersions(String,String)
+	 */
+	@Test
+	public void matchRequiredVersions_shouldAllowRequiredVersionWithPlus() {
+		String openmrsVersion = "1.4.3";
+		String requiredOpenmrsVersion = "1.4.2+";
+		Assert.assertTrue(ModuleUtil.matchRequiredVersions(openmrsVersion, requiredOpenmrsVersion));
```

Comment:
Please test the edges. So in this case values such as: "1.4.1", "1.4.1.99", "1.4.2", "1.4.3", "1.5.0", ... etc.
You don't need to reassign `openmrsVersion` each time then, I'm good with something like:
```java
String requiredOpenmrsVersion = "1.4.2+";
Assert.assertFalse(ModuleUtil.matchRequiredVersions("1.4.1", requiredOpenmrsVersion));
Assert.assertTrue(ModuleUtil.matchRequiredVersions("1.4.2", requiredOpenmrsVersion));
Assert.assertTrue(ModuleUtil.matchRequiredVersions("1.4.3", requiredOpenmrsVersion));
...
```